src/odemis/acq/stream/_helper.py

```diff
@@ -93,6 +93,25 @@ def __init__(self, name, detector, dataflow, emitter, scanner=None, sstage=None,
         #  * Rep set: Rep (as requested) + ROI (current) → PxS (updated)
         #    The repetition is adjusted to fit the hardware limits
 
+        # If "exposureTime" not detvas and detector has VA "exposureTime",
```

Comment:
It should be possible to make that comment less identical to the code it's supposed to document. How about: "If no local exposureTime VA is requested, automatically provide an integrationTime VA, which allows longer exposure by doing image integration."